src/Objects/ObjectBase.php

```diff
@@ -57,6 +57,20 @@ abstract class ObjectBase
      */
     public $localeAlternate = [];
 
+    /**
+     * The price amount specified.
+     *
+     * @var null|string
+     */
+    public $priceAmount;
```

Comment:
Shouldn't this property be a float type instead? [Documentation](https://developers.facebook.com/docs/payments/product/) says that this is a decimal number.